src/gluonts/model/seq2seq/_seq2seq_network.py

```diff
@@ -89,7 +89,7 @@ def compute_decoder_outputs(
         encoder_output_static, encoder_output_dynamic = self.encoder(
             scaled_target, embedded_cat, past_feat_dynamic_real
         )
-        decoder_input_static, _, decoder_input_dynamic = self.enc2dec(
+        decoder_input_static, decoder_input_dynamic = self.enc2dec(
```

Comment:
This must have been a pretty big bug in the `seq2seq` library right? If you look at the default `enc2dec` the third second output is the decoder_input_dynamic, not the third, thus decoder_input_dynamic has been dropped thus far and future_dynamic_feat used in its place

Reply:
I also removed the third output of `enc2dec` because it was never used and I did not see the point, but I'm open to putting that back.

Reply:
@lostella do you have any preference here regarding my last comment?